### PR TITLE
Hide print button in orders listing page

### PR DESCRIPTION
**Pull request by GytisZum**
image
image
image

### CODE REVIEW

File: controllers/admin/AdminDPDBalticsSettingsController.php [hunk 1]
```diff
@@ -114,6 +114,21 @@ protected function initOptions()
                     'title' => $this->l('Save'),
                 ],
             ],
+            'orders_page_configuration' => [
+                'title' => $this->l('Orders listing page configuration'),
```
Comment (tomjas1997) [inline controllers/admin/AdminDPDBalticsSettingsController.php:118]: missing FILE_NAME and use module. 

$this->module->l('Save', self::FILE_NAME),

File: dpdbaltics.php [hunk 2]
```diff
@@ -1267,6 +1273,7 @@ public function hookActionOrderGridDefinitionModifier(array $params)
 
         $definition = $params['definition'];
 
+        if (!(bool) Configuration::get(Config::HIDE_ORDERS_LABEL_PRINT_BUTTON)) {
```
Comment (tomjas1997) [inline dpdbaltics.php:1276]: Allowing bulk actions, but removed singular?

Comment (GytisZum) [reply dpdbaltics.php:1276]: We got approvement that we hide button only and keeping bulk actions

File: src/Install/Installer.php [hunk 3]
```diff
@@ -220,6 +220,7 @@ public function getDefaultConfiguration()
             Config::IMPORT_LINES_SKIP => 1,
             Config::IMPORT_FIELD_SEPARATOR => ';',
             Config::IMPORT_DELETE_OLD_DATA => 0,
+            Config::HIDE_ORDERS_LABEL_PRINT_BUTTON => 0,
```
Comment (tomjas1997) [inline src/Install/Installer.php:223]: I don't see it in upgrade script

**Review by tomjas1997 — APPROVED** [review]

**Pull request merged**